oelint_adv/color.py

```diff
@@ -1,10 +1,30 @@
-_COLOR = None
+from colorama import Fore
+from colorama.ansi import AnsiCodes
 
+_COLOR = False
+_COLORS_BY_SEVERITY = {
+    "neutral": Fore.WHITE,
```

Comment:
TBH I really don't like the ides of that "neutral" color and I will not merge it like this

The whole operation can be done with
```
_COLORS = {
    'info': Fore.GREEN,
    'warning': Fore.YELLOW,
    'error': Fore.RED,
}

_color = _COLORS.get(_severity, '')
_style = Style.RESET_ALL
return [(_line, f'{_path}:{_color}{_line}:{_severity}:{_display_id}:{override_msg}{style}')]
```

I don't see the need to make it more complicated than this